ext/exif/exif.stub.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/** @return string|false */
+function exif_tagname(int $index) {}
+
+/** @return array|false */
+function exif_read_data($filename, ?string $sections_needed = null, bool $sub_arrays = false, bool $read_thumbnail = false) {}
```

Comment:
`$sections_needed` can't be null. It probably should be made nullable -- feel free to do that.

Reply:
Like this?
https://github.com/php/php-src/pull/4530/commits/754d84fed586feb24307c115749a304388461cb3

Reply:
Yup!